src/02-data-types.js

```diff
@@ -36,7 +40,9 @@ function convertToNumber(val) {}
  *  convertToString(99);
  *  //> "99"
  */
-function convertToString(val) {}
+function convertToString(val) {
+  return String(val)
```

Comment:
I'm actually a big fan of this way of execution.  Remember we can do this in other ways.  Try to think of another method.